### ISSUE TITLE
IndexedDB: Operation never resolves with multiple store instances when DB and stores already exist

### ISSUE THREAD

**Issue by jpark3000** [mozilla/localForage#512]
I am experiencing an issue in Chrome Version 47.0.2526.111 (64-bit) AND Firefox 42 with both the 1.3.2 release and a package built from the latest commit (2f19724) where if I have an existing DB in `IndexedDB` with multiple stores, then create multiple instances to those stores, and then try to perform operations on those instances; only the last defined instance's operations will resolve.

Example:
```
// These already exist in IndexedDB
var store1 = localforage.createInstance({name: 'db', storeName: 'store1'});
var store2 = localforage.createInstance({name: 'db', storeName: 'store2'});

store1.setItem('dog', 'bark').then(function(value) {
  // Never outputs
  console.log(value);
}, function(error) {
  // Never outputs
  console.log(error);
};

store2.setItem('cat', 'meow').then(function(value) {
  // Will resolve and set value in IndexedDB
  console.log(value);
}, function(error) {
  console.log(error);
};
```

If `IndexedDB` is completely empty and I run the above script, then both operations will resolve successfully. 

Please let me know if you need any more info or if I am just missing something completely obvious.

Thanks!

**Comment by sebweaver** [comment]
It's because the dist files, reflecting the change https://github.com/mozilla/localForage/commit/2f197241de1039da9569295bf3c878ec0f6c2bdf, have not been committed yet.

You should wait for release 1.3.3 which would include this patch.

**Comment by tofumatt** [comment]
Just released version 1.3.3 so this should be fixed.
